lib/src/base-styles/_nx-form-select.scss

```diff
@@ -11,7 +11,7 @@
   -webkit-appearance: none;
   -moz-appearance: none;
   appearance: none;
-  background: #fff url("data:image/svg+xml;utf8, \
+  background: var(--rsc-component-background-color) url("data:image/svg+xml;utf8, \
```

Comment:
I'm not sure that we would want this, the text input, or the policy slider to also be tied to this variable. I think a form element background colour variable might be a better way to go.

Reply:
> I'm not sure that we would want this, the text input, or the policy slider to also be tied to this variable. I think a form element background colour variable might be a better way to go.

aa490bf3